### DISCUSSION TITLE
staying-out-of-net-open-source-dependency-hell/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#38] [Blog comments]
# staying-out-of-net-open-source-dependency-hell/

https://blogs.taiga.nl/martijn/posts/staying-out-of-net-open-source-dependency-hell/

**Comment by disqus-comments-migrator** [comment]
<em>Matt Hinze commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at april 26 2010, 07:24</em>

---

<p>I agree.  Sometimes copy paste is the best approach.  Dependency is risk, and this is just one of many ways to manage it.</p>
